app/models/user.rb

```diff
@@ -517,6 +517,14 @@ def drafts
     .order('created DESC')
   end
 
+  def notes_for_tags(tagnames)
+    Node.includes(:node_tag, :tag)
+        .where('term_data.name IN (?)', tagnames)
```

Comment:
Hi @toshitapandey, can you make this addition? We need to search by the user in the request params. Thanks!
```suggestion
        .where('term_data.name IN (?)', tagnames)
        .where(uid: uid)
```

Reply:
@TildaDares - Updated the query. Please take a look.

Reply:
@TildaDares - please take a look.